tests/test_visualization.py

```diff
@@ -59,19 +59,19 @@ def _checkButtonAndMethod(buttonName, button, method, *args, **kwargs):
     _compareHalvesOfLog(buttonName, method)
 
 
-def test_DetectorPanel():
+def test_DetectorPlotView():
     """
     Test base class for all detector panels.
     
     Aim: To check the buttons are mapped to the correct functions.
     """
 
     # check we can initialise
-    q, w = _initWindow(vis.DetectorPanel, TEST_FILE_1D)
+    q, w = _initWindow(vis.DetectorPlotView, TEST_FILE_1D)
 
     # check buttons can be pressed and don't cause error and give same log output as the methods they should be attached to
     #_checkButtonAndMethod(buttonName, button, method, *argsForMethod, **kwargsForMethod)
-    _checkButtonAndMethod("modalPlotButton", w.modalPlotButton, w.modalPlotButtonClicked, 0)
+    _checkButtonAndMethod("modalPlotButton", w.modalFocusButton, w.modalFocusButtonClicked, 0)
```

Comment:
Propose to comment these button checks out or remove them altogether just now (modalFocusButton to lotEnergyButton) since this DetectorPlotView doesn't have these buttons anymore and to avoid errors during testing. Should add these tests back in when checking the actual pop-out window though.